platform/analysis-impl/src/com/intellij/codeInspection/ex/HTMLComposerImpl.java

```diff
@@ -79,6 +79,11 @@ private void appendLocation(@NotNull StringBuilder buf, final RefElement refElem
       buf.append(NBSP);
       appendElementReference(buf, refElement, false);
     }
+    else if (refElement instanceof RefDirectory){
+      buf.append("dir");
```

Comment:
Please add `inspection.export.results.directory` in analysis bundle (`community/platform/analysis-api/resources/messages/AnalysisBundle.properties`)